.gitignore

```diff
@@ -27,6 +27,5 @@ build/Release
 node_modules
 
 # Typescript Specific
-typings/
```

Comment:
Looks like this solved the flakiness issue, but make sure to keep that commit separate from the next prop name generation.